infrastructure/terraform/bin/terratest.sh

```diff
@@ -22,7 +22,7 @@ if ! [ -x "$(command -v pip)" ] && [ -x "$(command -v pip3)" ]; then
   pip3 install --upgrade pip
 fi
 
-pip install terraform-compliance==1.3.48 --upgrade --quiet  # always upgrade to latest version
```

Comment:
Pipeline was failing - we're using terraform 1.10.0 -  support for this was introduced in terraform-compliance 1.3.49